app/views/clients/index.html.slim

```diff
@@ -1,4 +1,4 @@
-= will_paginate
+= will_paginate @clients, renderer: BootstrapPagination::Rails
```

Comment:
maybe with the new gem is needed, but the will_paginate doesn't require the name of the collection if one is in the same controller. That's why it was not included here but included in the other will_paginate calls.